AndroidSDK/src/com/leanplum/internal/Util.java

```diff
@@ -723,9 +724,14 @@ public static boolean isConnected() {
     return CollectionUtil.uncheckedCast(current);
   }
 
-  public static <T> void executeAsyncTask(AsyncTask<T, ?, ?> task, T... params) {
+  public static <T> void executeAsyncTask(boolean singleThread, AsyncTask<T, ?, ?> task,
```

Comment:
maybe make convenience methods, that way its harder to not confuse them?
e.g. `executeSingleThreadAsyncTask`